build/workspace.bzl

```diff
@@ -80,23 +73,23 @@ def debian_image_dependencies():
         container_pull(
             name = "debian-base-" + arch,
             architecture = arch,
-            digest = _DEBIAN_BASE_DIGEST,
+            digest = "sha256:8ccb65cd2dd7e0c24193d0742a20e4a673dbd11af5a33f16fcd471a31486866c",  # 0.4.1
```

Comment:
why did the digest change but not the version comment?

Reply:
/hold
It's because these images are actually multiplatform manifests, and bazel wants to refer to the platform specific one.

Let me see if I can figure out a better way to get the digest for each arch correctly, with this change we'll always download the linux amd64 one

Reply:
See `docker run mplatform/mquery busybox`

Reply:
Done